README.md

```diff
@@ -1,8 +1,8 @@
 # Sui README
 
-![Sui icon](doc/static/Sui_Icon_Brand.png)
-
-Welcome to Sui, a next generation smart contract platform with high throughput, low latency, and an asset-oriented programming model powered by the [Move](https://github.com/MystenLabs/awesome-move) programming language! Find everything you need within the [Sui Developer Guides](doc/src/learn/index.md) and supporting materials.
+|             |             |
+| ----------- | ----------- |
```

Comment:
```suggestion
```

The first row in the table looks a bit funky: 
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/3513142/159553230-38a3957c-cc20-4da3-bf65-7754a64c4e22.png">

Think this will fix it, but not sure.

Reply:
Sadly, that doesn't seem to work as that row appears to be the delimiter between table headings and regular rows. I removed it, and it rendered as:

| | | | Welcome to Sui, a next-generation smart contract platform with high throughput, low latency, and an asset-oriented programming model powered by the [Move programming language](https://github.com/MystenLabs/awesome-move)! Find everything you need within the [Sui Developer Guides](https://github.com/MystenLabs/sui/blob/Clay-Mysten-patch-13/doc/src/learn/index.md) and supporting materials below. |

I tried removing the top, empty row earlier but get this instead:

| ----------- | ----------- | | Welcome to Sui, a next-generation smart contract platform with high throughput, low latency, and an asset-oriented programming model powered by the [Move programming language](https://github.com/MystenLabs/awesome-move)! Find everything you need within the [Sui Developer Guides](https://github.com/MystenLabs/sui/blob/Clay-Mysten-patch-13/doc/src/learn/index.md) and supporting materials below. |

Looks like the heading is required by most parsers:
https://stackoverflow.com/questions/17536216/create-a-table-without-a-header-in-markdown